db/migrations/versions/035_drop_geospatial_data_blob.py

```diff
@@ -0,0 +1,44 @@
+"""drop geospatial data_blob
+
+Revision ID: 035_drop_geospatial_data_blob
+Revises: 034_geospatial_itl1_region_name
+Create Date: 2024-05-14 16:00:44.560549
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "035_drop_geospatial_data_blob"
+down_revision = "034_geospatial_itl1_region_name"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("geospatial_dim", schema=None) as batch_op:
+        batch_op.drop_column("data_blob")
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("geospatial_dim", schema=None) as batch_op:
+        batch_op.add_column(
+            sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), autoincrement=False, nullable=True)
```

Comment:
Not sure it matters but original migration (no. 22) defined this column without `autoincrement` argument: `sa.Column("data_blob", postgresql.JSONB(astext_type=sa.Text()), nullable=True)`

Reply:
Hmmm good catch, that bit was auto-generated by the alembic migration! I'll try removing it and see if it thinks there are unapplied changes/anything breaks